File: CUSTOR.EICOnline.Angular/src/app/model/service-prerequisite.ts
export class ServicePrerequisite {
  public ServicePrerequisiteId: number;
  public DocumentId?: number;
  public IsDocument?: number;
  public IsActive: boolean;
  public ServiceId: number;
  public Description: string;
  public BusinessType: number;
  public DescriptionEnglish: string;
  public ServiceNameEnglish: string;
  public upload?: boolean;
  public URL?: string;

  constructor(servicePrerequisiteId?: number, isActive?: boolean, description?: string, DescriptionEnglish?: string, serviceId?: number, ServiceNameEnglish?: string, BusinessType?: number) {
    this.ServicePrerequisiteId = servicePrerequisiteId;
    this.ServiceId = serviceId;
    this.Description = description;
    this.DescriptionEnglish = DescriptionEnglish;
    this.IsActive = isActive;
    this.ServiceNameEnglish = ServiceNameEnglish;
    this.BusinessType = BusinessType;
  }

  // public ServiceNameEnglish: string;
}

File: CUSTOR.EICOnline.DAL/EntityLayer/ServicePrerequisite.cs
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using CUSTOR.EICOnline.DAL.EntityLayer;

namespace CUSTOR.EICOnline.DAL
{
  public partial class ServicePrerequisite
  {
    public ServicePrerequisite()
    {
      Document = new HashSet<Document>();
    }

    [Key]
    [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
    public int ServicePrerequisiteId { get; set; }

    public string Description { get; set; }

    public string DescriptionEnglish { get; set; }

    public int ServiceId { get; set; }

    public int BusinessType { get; set; }

    public bool IsActive { get; set; }
    public bool IsDocument { get; set; }

    [NotMapped]
    public string ServiceNameEnglish { get; set; }

    [ForeignKey(name: "ServiceId")]
    public Service Service { get; set; }

    public ICollection<Document> Document { get; set; }
  }

  public partial class ServicePrerequisiteLookup
  {
    public ServicePrerequisiteLookup()
    {
    }

    public int? ServicePrerequisiteId { get; set; }

    public string Description { get; set; }

    public string DescriptionEnglish { get; set; }

    public int ServiceId { get; set; }

    public int BusinessType { get; set; }
    public bool IsActive { get; set; }

    [NotMapped]
    public string ServiceNameEnglish { get; set; }

    public Service Service { get; set; }
    public ICollection<Document> Documents { get; set; }
  }
}